test/HorizontalHeatmapTests.js

```diff
@@ -1,21 +1,13 @@
 /* eslint-env mocha */
-import {
-  configure,
-  // render,
-} from 'enzyme';
-
+import Enzyme from 'enzyme';
```

Comment:
Just curious, why this change?

Reply:
No reason, really. I just found it easier to figure out where the setup/teardown code came from.